### PR TITLE
[Process] add tests for php executable finder if file does not exist

### PR DESCRIPTION
**Pull request by ahmedash95**
| Q             | A
| ------------- | ---
| Branch?       | master?
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | 
| License       | MIT
| Doc PR        | 

regarding Process component, I just noticed there is no test covers the case when the PHP binary is not valid so I added the test to make sure it works as expected also it increases the coverage of the process component

### CODE REVIEW

**Review by darielopper — APPROVED** [review]

File: src/Symfony/Component/Process/Tests/PhpExecutableFinderTest.php [hunk 1]
```diff
@@ -46,4 +46,19 @@ public function testFindArguments()
             $this->assertEquals($f->findArguments(), [], '::findArguments() returns no arguments');
         }
     }
+
+    /**
+     * tests find() with invalid file.
+     */
```
Comment (fabpot) [inline src/Symfony/Component/Process/Tests/PhpExecutableFinderTest.php]: The phpdoc should be rmoved

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Process/Tests/PhpExecutableFinderTest.php (hunk 1 above)
Comment (ahmedash95) [reply src/Symfony/Component/Process/Tests/PhpExecutableFinderTest.php]: Done 👍

**Review by nicolas-grekas — APPROVED** [review]
(for 3.4)

**Comment by nicolas-grekas** [comment]
Thank you @ahmedash95.

**Pull request merged**